stacks-signer/src/chainstate.rs

```diff
@@ -0,0 +1,375 @@
+use blockstack_lib::chainstate::nakamoto::NakamotoBlock;
```

Comment:
Would you be able to run cargo clippy on stacks-signer please? There are a few suggestions:

`cargo clippy -p stacks-signer --no-deps --tests --all-features`

Reply:
Sure -- done in fdcaea8b91972ad721d9d9e0579ac89fb531ddc0